src/components/fundingCard.tsx

```diff
@@ -7,69 +7,92 @@ import { Card } from "@/components/ui/card";
 import { ProductResponseDTO } from "@team-golfslag/conflux-api-client/src/client";
 import { ExternalLink } from "lucide-react";
 import { Badge } from "@/components/ui/badge.tsx";
+import { ProjectOrganisationResponseDTO } from "@team-golfslag/conflux-api-client/src/client.ts";
+import { isValidUrl } from "@/utils/urlUtils.ts";
 
-type CardProps = { product: ProductResponseDTO };
+type CardProps = {
+  product?: ProductResponseDTO;
+  organisation?: ProjectOrganisationResponseDTO;
+};
```

Comment:
ik bedoelde meer dit
```suggestion
type CardProps = {
  object: ProductResponseDTO | ProjectOrganisationResponseDTO;
};
```